src/components/shared/Paragraph.tsx

```diff
@@ -13,15 +13,26 @@ export default function Paragraph({
 }) {
   const CHARACTER_LIMIT_EXCEEDED =
     (characterLimit && description.length > characterLimit) ||
-    description.split('\n').length > 1
+    description.split('\n').length > 3
```

Comment:
One thing to possibly consider is if they have one super long line of text. I wonder if we should expand this to check each line for a character limit, wdyt?

Something like:
```ts
const isCharacterLineExceeded(description: string, characterLimit: number | undefined ) {
  if (!characterLimit) return false
  const lines = description.split('\n')
  if (lines > 3) return false
  return lines.every(line => line.length < characterLimit)
}
...
const CHARACTER_LIMIT_EXCEEDED = isCharacterLineExceeded(description, characterLimit)
```

Reply:
@wraeth-eth Not sure I follow this. If one line is super long then it'll just get cut off by the characterLimit right?

Reply:
Yeah I think you're right - resolving